nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -143,31 +173,50 @@ private boolean lookupAndBuildCache(Symbol.ClassSymbol klass) {
         LOG(DEBUG, "DEBUG", "Found source of class: " + className + ", jar: " + jarPath);
         // Avoid reloading for classes w/o any stubs from already loaded jars.
         if (!loadedJars.contains(jarPath)) {
-          JarFile jar = juc.getJarFile();
-          if (jar == null) {
-            throw new Error("Cannot open jar: " + jarPath);
-          }
-          loadedJars.add(jarPath);
-          JarEntry astubxJE = jar.getJarEntry(DEFAULT_ASTUBX_LOCATION);
-          if (astubxJE == null) {
-            LOG(VERBOSE, "Warn", "Cannot find jarinfer.astubx in jar: " + jarPath);
+          String jarName = FilenameUtils.getBaseName(jarPath).replaceFirst("\\.[ja]ar-abi$", "");
```

Comment:
Again, our mapping might need to be more general. Perhaps the solution is to just load the models eagerly after all.